app/grandchallenge/components/form_fields.py

```diff
@@ -111,6 +111,8 @@ def get_image_field(self):
             current_value = Image.objects.get(pk=self.initial)
         elif UserUpload.objects.filter(pk=self.initial).exists():
             current_value = UserUpload.objects.get(pk=self.initial)
+        elif self.initial:
```

Comment:
There is always the case where `self.initial` is not set (None), for example, in create views, and therefore the last conditions should check that the value is there with `elif` instead of `else`

Reply:
outdated now